opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizEventRepository.java

```diff
@@ -21,4 +25,37 @@ public boolean hasEvent(@NonNull String baseEntityId, @NonNull String eventType)
         }
         return hasEvent;
     }
+
+    public void ReprocessClients()
```

Comment:
:smile: `camelCase` please and let's rename this to `processSkippedClients` or something a bit more descriptive enough that only unprocessed clients are being processed here